test/system/src/main/java/io/pravega/test/system/framework/Utils.java

```diff
@@ -89,23 +89,26 @@ public static Service createBookkeeperService(final URI zkUri) {
         }
     }
 
-    public static Service createPravegaControllerService(final URI zkUri, String serviceName) {
+    public static Service createPravegaControllerService(final URI zkUri, String serviceName, boolean secure) {
         switch (EXECUTOR_TYPE) {
```

Comment:
`isSecure` or `enableSecurity`, etc.

Also, curious about this: In this PR you seem to be adding TLS only (as auth is already there). So, this variable and the same variable in other classes in this PR might be referring to TLS only. So, wouldn't it make sense to just call it `enableTls` or the like. Otherwise, people will be misled to believe its setting `Auth` too.